site/src/sphinx/client-java.rst

```diff
@@ -565,15 +565,15 @@ client profile JAR again.
 Spring Boot integration
 -----------------------
 If you are using `Spring Framework <https://spring.io/>`_, you can inject :api:`com.linecorp.centraldogma.client.CentralDogma`
-client very easily. First, add ``centraldogma-client-spring-boot-starter`` into your dependencies.
+client very easily. First, add ``centraldogma-client-spring-boot3-starter`` into your dependencies.
```

Comment:
Comment by @minwoox 

What do you think of also specifying `centraldogma-client-spring-boot2-starter` so that users who are using `spring-boot2` or jdk < 17 are aware of such an option

Reply:
A preview on how this block will be rendered

![Screen Shot 2023-04-12 at 11 55 26 AM](https://user-images.githubusercontent.com/8510579/231336566-d71f64d4-f049-4e4e-b029-425a592d978d.png)